packages/common/test/events/edit.test.ts

```diff
@@ -352,6 +353,33 @@ describe("HubEvents edit module", () => {
     });
   });
 
+  describe("deleteHubEvent", () => {
+    it("calls deleteEvent", async () => {
+      const authdCtxMgr = await ArcGISContextManager.create({
+        authentication: MOCK_AUTH,
+        currentUser: {
+          username: "casey",
+        } as unknown as PortalModule.IUser,
+        portal: {
+          name: "DC R&D Center",
+          id: "BRXFAKE",
+          urlKey: "fake-org",
+        } as unknown as PortalModule.IPortal,
+        portalUrl: "https://myserver.com",
+      });
+      const deleteEventSpy = spyOn(eventsModule, "deleteEvent").and.callFake(
+        () => {
+          return Promise.resolve();
+        }
+      );
+      await deleteHubEvent("0o1", authdCtxMgr.context.userRequestOptions);
```

Comment:
We should be passing `authdCtxMgr.context.hubRequestOptions` here and update the `expect` statement below to reflect that